consul_kv_migrator.py

```diff
@@ -0,0 +1,97 @@
+from consul_kv import Connection
+import json
+import ast
+from prettytable import PrettyTable
+import getopt
+import sys
```

Comment:
We should import libraries in systemtic format .. so system libraries should be imported first .. then app lib and third party library and so on

Reply:
Done with changes